charts/kopf/.test.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+trap "exit 1" TERM
+export TOP_PID=$$
+
+export project_name="kopf-$(date +'%d%m%Y')"
+
+install() {
+  echo "install - $(pwd)"
+
+  oc new-project ${project_name}
+  helm install kopf kopf
```

Comment:
is this referencing the local directory chart? or a remote? I've always done `helm install {name} .` (dot for current dir)

Reply:
Yepp. Just the local chart. Just didn't know where this gets run from so just assumed the use of the dir name. Went ahead and fixed this.